catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/history/HistoryResponse.java

```diff
@@ -13,22 +13,36 @@
  */
 package org.codice.ddf.catalog.ui.metacard.history;
 
+import java.io.Serializable;
 import java.time.Instant;
 import java.util.Date;
+import java.util.List;
+import java.util.Map;
 
 public class HistoryResponse {
   private Instant versioned;
 
-  private String id;
+  private Map<String, List<Serializable>> historyMetacardAttributes;
```

Comment:
✏️ It's redundant to have `historyMetacardAttributes` in a `HistoryResponse`

Reply:
❓ Won't this be serialized oddly if the value is a `List<Serializable>`? Every attribute that is not multi valued will appear to be multi valued.

Reply:
What's the alternative? The only one I can see is to just list out every single attribute. 
To you second comment, that is true but if I don't do this, then I have to separate out the two types of fields that are multivalued or not.